x/assets/keeper/params.go

```diff
@@ -39,22 +39,22 @@ func (k Keeper) GetParams(ctx sdk.Context) (*assetstypes.Params, error) {
 	return ret, nil
 }
 
-func (k Keeper) GetExocoreLzAppAddress(ctx sdk.Context) (common.Address, error) {
-	depositModuleParam, err := k.GetParams(ctx)
+func (k Keeper) GetExocoreGatewayAddress(ctx sdk.Context) (common.Address, error) {
+	param, err := k.GetParams(ctx)
 	if err != nil {
 		return common.Address{}, err
 	}
-	return common.HexToAddress(depositModuleParam.ExocoreLzAppAddress), nil
+	return common.HexToAddress(param.ExocoreLzAppAddress), nil
```

Comment:
can we rename this **ExocoreLzAppAddress** to ExocoreGatewayAddress which is easy to understand? 

Additionally, If in the future **exocoreBtcGateway** and **exocoreSolGateway**  etc. cannot be integrated with **exocoregateway**, how should this part of the logic be handled?

Reply:
Yeah, it should be renamed to ExocoreGatewayAddress. However, the renaming might cause inconsistency in the data within the KVStore, so we need to address these issues together at an appropriate time. I have created an issue to document these concerns
https://github.com/ExocoreNetwork/exocore/issues/113